datadog-agent-nvml.yaml

```diff
@@ -0,0 +1,122 @@
+package:
+  name: datadog-agent-nvml
+  # This package has two git checkouts. For each new release, the commit SHA for
+  # DataDog/integrations-core must also be updated.
+  version: 1.0.9
+  epoch: 0
+  description: "Checks NVIDIA Management Library (NVML) exposed metrics through the Datadog Agent and can correlate them with the exposed Kubernetes devices"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - datadog-agent # always pick the latest agent to run the `integration install` c ommand.
+      - datadog-agent-core-integrations
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - datadog-agent
+      - datadog-agent-core-integrations
+      - rsync
+
+vars:
+  DD_HOME: / # agent being run by root expects /.
+  DD_SHARED: /usr/share/datadog-agent
+  DD_CONF: /etc/datadog-agent/conf.d
+
+pipeline:
+  # This integration wheel comes from the integrations-extras repository
+  - uses: git-checkout
+    with:
+      repository: https://github.com/DataDog/integrations-extras
+      tag: nvml-${{package.version}}
+      expected-commit: d38c5cdb4ab4d07f4432afb25e0ccd70341efb51
+  # The integrations requirements file comes from the integrations-core repository
+  # The `agent integration install` command expects it.
+  # TODO: ship it with the datadog-agent-integrations-core subpackage.
+  - uses: git-checkout
+    working-directory: /home/integrations-core
```

Comment:
This is still needed unless we provide the `integrations-core/datadog_checks_downloader` Python package with the `datadog-agent-integrations-core` package.